conans/test/functional/command_v2/help_test.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+
+from conans.client.tools import environment_append, save, six
+from conans.test.utils.tools import TestClient
+
+
+@unittest.skipIf(six.PY2, "v2.0: Only testing for Python 3")
```

Comment:
Probably both the skipIf and the run() method can be put in a common class and all command_v2 tests use it.